src/traces/bar/style.js

```diff
@@ -143,14 +147,26 @@ function getFontValue(attributeDefinition, attributeValue, index, defaultValue)
     var familyValue = helpers.getValue(attributeValue.family, index);
     var sizeValue = helpers.getValue(attributeValue.size, index);
     var colorValue = helpers.getValue(attributeValue.color, index);
+    var weightValue = helpers.getValue(attributeValue.weight, index);
+    var styleValue = helpers.getValue(attributeValue.style, index);
+    var stretchValue = helpers.getValue(attributeValue.stretch, index);
+    var variantValue = helpers.getValue(attributeValue.variant, index);
 
     return {
         family: helpers.coerceString(
           attributeDefinition.family, familyValue, defaultValue.family),
         size: helpers.coerceNumber(
           attributeDefinition.size, sizeValue, defaultValue.size),
         color: helpers.coerceColor(
-          attributeDefinition.color, colorValue, defaultValue.color)
+          attributeDefinition.color, colorValue, defaultValue.color),
+        weight: helpers.coerceColor(
```

Comment:
Why `coerceColor`, should this be `coerceString`? (Here and 3 below)

Reply:
Addressed by 7d3facf.